src/basic/parse-util.c

```diff
@@ -732,3 +733,23 @@ int parse_loadavg_fixed_point(const char *s, loadavg_t *ret) {
 
         return store_loadavg_fixed_point(i, f, ret);
 }
+
+static const char* const pcr_number_table[_PCR_MAX] = {
+        [PCR_CORE_FM]     = "corefirmware",
+        [PCR_PLAT_CONFIG] = "platformconfig",
+        [PCR_EXT_CODE]    = "extcode",
+        [PCR_EXT_FM]      = "extfirmware",
+        [PCR_BOOT_LD]     = "bootloader",
+        [PCR_GPT_PT]      = "gptpartition",
+        [PCR_POWER_ST]    = "powerstate",
+        [PCR_SECURE_BT]   = "secureboot",
+        [PCR_LINUX_KN]    = "linuxkernel",
+        [PCR_IMA_ST]      = "imastate",
+        [PCR_SYSTEMD_KN]  = "systemdkernel",
+        [PCR_SYSTEMD_BT]  = "systemdboot",
+        [PCR_SYSTEMD_STB] = "systemdstub",
+        [PCR_SHIM_CT]     = "shimcerts",
+        [PCR_LUKS_KEY]    = "lukskey",
```

Comment:
please do not abbreviate unnecessarily.

please make sure that that the right-hand-side matches the name of the left-hand-side, except for maybe a common prefix. i.e. if the string is called "foo" or "bar" then the enum should be called XYZ_FOO or XYZ_BAR, but certainly not XYZ_F or XYZ_B or so.

Also, we usually use "-" as separator for enum identifiers.

Reply:
> please do not abbreviate unnecessarily.
> 
> please make sure that that the right-hand-side matches the name of the left-hand-side, except for maybe a common prefix. i.e. if the string is called "foo" or "bar" then the enum should be called XYZ_FOO or XYZ_BAR, but certainly not XYZ_F or XYZ_B or so.
> 
> Also, we usually use "-" as separator for enum identifiers.

Thank you sir. Sorry I'm not clear on the "-" separator. Do you mind showing an example? Thank you.